impedance_force_node:
  ros__parameters:
    publish_rate_hz: 500.0

    world_name: foot_test
    model_name: rl12dof_urdf
    link_name: Foot_V3_Foot_v1_1

    force:
      x: 0.0
      y: 0.0
      z: 200.0

    timing:
      settle_time: 0.0
      duration: 1.0
